src/iop/filmicrgb.c

```diff
@@ -443,7 +447,7 @@ static inline void convert_to_spline_v3(dt_iop_filmicrgb_params_t* n)
 int legacy_params(dt_iop_module_t *self, const void *const old_params, const int old_version, void *new_params,
                   const int new_version)
 {
-  if(old_version == 1 && new_version == 5)
+  if(old_version == 1 && new_version == 6)
```

Comment:
in each section we need to explicitly add:

`enable_highlight_reconstruction = TRUE;`